CONTRIBUTING.md

```diff
@@ -30,6 +30,14 @@ since they may assume that you're still working on it.\
 _Desired behaviour:_ Write a message saying roughly the following "The CI
 failure is unrelated", so that the maintainers will fix it for you.
 
+_Event:_ The CI fails to build due to a formatting issue, but you can't do this
+on your computer.\
+_Desired behavior_: Please make sure you have read the following "Formatting"
+section so you can even handle this prior to commit your changes. This can be a
+special case since veterans will think it is your fault. However, you can always ask
+them for help. It's not a guilt asking for any technical help in conforming this policy.
+Just ask.
```

Comment:
This is technically not a _handling review_, because the formatter will start to fail before you even send something, since it'll warn that it doesn't understand the options, because they are from nightly, so I'd probably just add a note after `Then to format` with the wording I posted on other comment (simply because it's _way shorter_)

Reply:
Got it. I just noticed this due to network latency. Give me a minute to think of it.